frontend/src/components/pages/TeamDashboard/Presentation.js

```diff
@@ -25,6 +25,8 @@ import {
 } from "recharts";
 import { getMinimumAndMaximumValue } from "./utils";
 
```

Comment:
You are importing 2 things from utils. Wouldn't it make sense to have 1 import statement instead of 2?